### PR TITLE
Update URI schemes

### PR DESCRIPTION
**Pull request by yuvallanger**
License: MIT
Signed-off-by: Yuval Langer <yuval.langer@gmail.com>

### CODE REVIEW

**Comment by Kubuxu** [comment]
@RichardLitt can you look at it.

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ IPFS is a filesystem:
 IPFS is a web:
 - can be used to view documents like the web
 - files accessible via HTTP at `http://ipfs.io/<path>`
-- browsers or extensions can learn to use `ipfs://` directly
+- browsers or extensions can learn to use the `ipfs:/` or `fs:/` protocol designators directly
```
Comment (deltab) [inline README.md]: 'Scheme' is the standard term: https://tools.ietf.org/html/rfc3986#section-3.1

Comment (RichardLitt) [inline README.md]: Let's go with that?

**Comment by RichardLitt** [comment]
Oh, yeah! This looks good to me. Thanks!

**Pull request merged**